### ISSUE TITLE
Training with log Mel-Spectrograms

### ISSUE THREAD

**Issue by pbansal5** [SeanNaren/deepspeech.pytorch#543]
What changes can be made to the model architecture so as to make it work with log Mel Features instead? I have been using normalised version of log mel specs to train the DeepSpeech model but seem to be stuck at around 10% CER/30%WER. 
Since the mel-spectrograms have 80  features/time stamp instead of 160, I changed the initial Conv Layer's stride to 1 instead of 2. That relieved some of the issue. But how do I get to 3% CER with mel-specs?

The experiments are done on LibriSpeech clean (100hr), and have used 1024 hidden size with 5 layer deep model.

**Comment by SeanNaren** [comment]
What is your reference for 3% CER?

**Comment by pbansal5** [comment]
Hi Sean, I am talking about the 3.307 % CER models you have in the pretrained models of the repo on LibriSpeech Clean
